Lib/bdb.py

```diff
@@ -515,10 +505,10 @@ def get_all_breaks(self):
     # to get a data structure representing a stack trace.
 
     def get_stack(self, f, t):
-        """Get a list of frames in the stack trace.
+        """Return a list of (frame, lineno) in a stack trace and a size.
 
-        Get a list of records for a frame and all higher (calling) and
-        lower frames, and the size of the higher part.
+        List starts with original calling frame, if there is one.
+        Size may be number of frames above or below f.
```

Comment:
For the size, the line 'if f is None: i = max()', but I don't see why f wouldn't be None here.  It could come in as None or the 'while f is not None:' loop changes it until it is None.  So, the size would either be 0 (if there isn't a stack) or the size of the stack - 1.  It wouldn't be just the above part or below part.  I realize I'm probably missing something, but I don't see it.  Thanks.